rust/src/nfs/log.rs

```diff
@@ -35,43 +35,43 @@ pub extern "C" fn rs_nfs_tx_logging_is_filtered(state: &mut NFSState,
     return 0;
 }
 
-fn nfs_rename_object(tx: &NFSTransaction) -> Json
+fn nfs_rename_object(tx: &NFSTransaction, js: &mut JsonBuilder)
+    -> Result<bool, JsonError>
```

Comment:
As this function either errors out or returns true, and the Ok(val) is never checked, its probably better as `Result<(), JsonError>`.

If you pulled this from any of my examples, oops, I had meant to get rid of them.

Reply:
My bad, I pulled it from a place in dns where it does serve a purpose. Fixed in #5023